docs/rfc/0002_kubectl_rw.md

```diff
@@ -0,0 +1,49 @@
+|                    |                                      |
+| -------            |--------------------------------------|
+| Feature            | kubectl-rw: a kind of kubectl plugin |
+| Status             | In-progress                          |
+| Date               | 2022-06-29                           |
+| Authors            | Xinyu                                |
+| RFC PR #           |                                      |
+| Implementation PR #|                                      |
+|                    |                                      |
+
+# **Summary**
+
+kubectl-rw: a kubectl-plugin to deploy and manage risingwave operator and DBs.
+
+# **Motivation**
+
+For convenience, we will design a command line tool that allows users to quickly manage the risingwave operator and DBs.
+
+For example, user can do the operations as follows:
+
+```shell
+
+kubectl rw init # deploy the operator in the kubernetes cluster
+
+kubectl rw create instance --meta-storage memory # create a risingwave db in the kubernetes cluster and use the memory storage for mete-node
+
+kubectl rw upgrade # upgrade the risingwave db to the latest version
+```
+### What commands will be supported (Continually updated)
+
+- init: deploy the operator.
```

Comment:
I think `install` might be more suitable. And we can also provide an `uninstall` option.

Reply:
good idea.